src/backend/expungeservice/record_creator.py

```diff
@@ -180,7 +180,8 @@ def _edit_charges(charges: Tuple[OeciCharge, ...], edits):
                         else:
                             charge_edits["disposition"] = None
                     elif key in ("date", "probation_revoked"):
-                        charge_edits[key] = datetime.date(datetime.strptime(value, "%m/%d/%Y"))
+                        if value is not "":
```

Comment:
```suggestion
                        if value:
```

Reply:
"" is falsey, so that doesn't work

Reply:
Wait

Reply:
Oh, right, I was dealing with falsiness issues elsewhere, but I did this for ... Self-documenting code!

Reply:
Oh, right, I was dealing with falsiness issues elsewhere, but I did this for ... Self-documenting code!

Reply:
We know it's a string from the types though. So if it's false it must be the empty string. I just don't like how it makes me pause upon review.

Reply:
Yep yep very well